lib/core/enums/amplitude_events.dart

```diff
@@ -1,6 +1,17 @@
 enum AmplitudeEvents {
```

Comment:
This is not gonna be scalable. We have to come up with a better more generic solution. If i would create an event for every feature it's gonna be a huge amount of work to keep track of these events

Reply:
Do you mean this from an flutter implementation perspective? A solution being some sort of link between flutter and amplitude so events planned in amplitude can be accessed without defining them. Like locals?
Or from a broader perspective to use more general events in amplitude as well?

Reply:
Yes, this way of doing it is not scalable. I was thinking about general events. Using locals is not a good idea as that's meant for translations anyway. Or a framework that would allow us to not implement/add so many custom events in a file but rather.